frontend/src/Editor/Container.jsx

```diff
@@ -541,7 +545,7 @@ export const Container = ({
       removeComponent,
       currentLayout,
       deviceWindowWidth,
-      selectedComponents,
+      selectedComponents: useEditorStore.getState().selectedComponents,
```

Comment:
Check whether it is being used in any of the components. If not we can remove this from `containerProps`